homeassistant/components/camera/demo.py

```diff
@@ -0,0 +1,39 @@
+"""
+homeassistant.components.camera.demo
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Demo platform that has a fake camera.
+"""
+import os
+from homeassistant.components.camera import Camera
+
+i = 0
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """ Sets up the Demo camera. """
+    add_devices([
+        DemoCamera('Demo camera')
+    ])
+
+
+class DemoCamera(Camera):
+    """ A Demo camera. """
+
+    def __init__(self, name):
+        super().__init__()
+        self._name = name
+
+    def camera_image(self):
+        """ Return a faked still image response. """
+        global i
+
+        i = 1 if i == 5 else i + 1
+        with open(('{}/ha{}.png'.format(
```

Comment:
Can you rename the images to be `demo_1.png` … `demo_5.png` ?

Can you use `os.path.join` to create file paths?

```python
image_path = os.path.join(os.path.dirname(__file__), 'demo_{}.png'.format(self.i))
```